src/containers/SearchPage/SearchContainer.jsx

```diff
@@ -0,0 +1,359 @@
+        <SearchPage
+          closeUrl="/#"
+          searchString={searchParams.query || ''}
+          onSearchFieldChange={e => this.updateQuery(e.target.value)}
+          onSearch={this.onQuerySubmit}
+          searchFieldPlaceholder={t('searchPage.searchFieldPlaceholder')}
+          onSearchFieldFilterRemove={this.onSearchFieldFilterRemove}
+          searchFieldFilters={activeSubjectsMapped}
+          activeFilters={activeSubjectsMapped}
+          onActiveFilterRemove={() => {}}
+          messages={searchPageMessages}
+          resourceToLinkProps={resourceToLinkProps}
+          filters={searchFilters}>
+          <SearchResults
+            results={results}
+            resultMetadata={resultMetadata}
+            filterState={this.state.searchParams}
+            enabledTabs={enabledTabs}
+            onTabChange={this.updateTab}
+          />
+          <Pager
+            page={
+              this.state.searchParams.page
+                ? parseInt(this.state.searchParams.page, 10)
+                : 1
+            }
+            lastPage={resultMetadata.lastPage}
+            query={this.state.searchParam}
+            pathname=""
+            onClick={this.updateFilter}
+            pageItemComponentClass="button"
+          />
+        </SearchPage>
+      </OneColumn>
+    );
+  }
+}
+
+SearchContainer.propTypes = {
+  location: shape({
+    search: string,
+  }).isRequired,
+  history: shape({
+    push: func.isRequired,
+  }).isRequired,
+  clearSearchResult: func.isRequired,
+  lastPage: number.isRequired,
+  results: arrayOf(ArticleResultShape).isRequired,
+  searching: bool.isRequired,
+  search: func.isRequired,
+  enabledTabs: arrayOf(
+    shape({
+      name: string,
+      value: string,
+      type: string,
+    }),
+  ),
+  subjects: arrayOf(SubjectShape),
+  resultMetadata: shape({
+    totalCount: number,
+    lastPage: number,
+  }),
+  filters: arrayOf(FilterShape),
+  match: shape({
+    params: shape({
+      subjectId: string,
+    }),
+  }),
+  locale: string.isRequired,
+};
+
+SearchContainer.defaultProps = {
+  enabledTabs: [
+    { value: 'all', name: 'all' },
+    { value: 'topic-article', type: 'context-types', name: 'subject' },
+    {
+      value: 'urn:resourcetype:subjectMaterial',
+      type: 'resource-types',
+      name: 'subject-material',
+    },
+    {
+      value: 'urn:resourcetype:learningPath',
+      type: 'resource-types',
+      name: 'learning-path',
+    },
+    {
+      value: 'urn:resourcetype:externalResource',
+      type: 'resource-types',
+      name: 'external-learning-resources',
+    },
+  ],
+  filters: [],
+  subjects: [],
+};
+
+const mapDispatchToProps = {
+  search: actions.search,
+  clearSearchResult: actions.clearSearchResult,
+  fetchSubjects: subjectActions.fetchSubjects,
+  fetchSubjectFilters: filterActions.fetchSubjectFilters,
+  fetchFilters: filterActions.fetchFilters,
+  fetchTopicsWithIntroductions: topicActions.fetchTopicsWithIntroductions,
```

Comment:
Some of these (clearSearchResult, fetchSubjectFilters, fetchTopicsWithIntroductions) are never called. Why?

Reply:
Removed, not needed in the new search as I can see.